### PR TITLE
CORE-1372: added the ability to reject a VICE access request after it…

### PR DESCRIPTION
**Pull request by slr71**
… has already been approved

### CODE REVIEW

File: src/terrain/services/requests.clj [hunk 1]
```diff
@@ -67,14 +67,38 @@
   "Marks a request as being in progress."
   (request-update-fn "Your request is in progress." "in-progress"))
 
-(def request-rejected
+(def mark-request-rejected
   "Marks a request as having been rejected."
   (request-update-fn "No deinal reason given." "rejected"))
```
Comment (ash3rz) [inline src/terrain/services/requests.clj:72]: Looks like a tpyo here (outside of your PR changes, but figured I'd mention it):
```suggestion
  (request-update-fn "No denial reason given." "rejected"))
```

**Review by ash3rz — APPROVED** [review]
LGTM 👍

File: src/terrain/services/requests.clj (hunk 1 above)
Comment (slr71) [reply src/terrain/services/requests.clj:72]: Ah, I completely missed that. Thanks for catching it.

**Comment by slr71** [comment]
Thanks for the review!

**Pull request merged**